evalml/data_checks/invalid_targets_data_check.py

```diff
@@ -38,9 +38,9 @@ def validate(self, X, y):
         null_rows = y.isnull()
         if null_rows.any():
             messages.append(DataCheckError("{} row(s) ({}%) of target values are null".format(null_rows.sum(), null_rows.mean() * 100), self.name))
-        valid_target_types = numeric_and_boolean_dtypes + categorical_dtypes
+        valid_target_types = [str(dtype) for dtype in numeric_and_boolean_dtypes + categorical_dtypes]
 
-        if y.dtype.name not in valid_target_types:
+        if y.dtype.name.lower() not in valid_target_types:
```

Comment:
How come we need this change?

Reply:
Ha, IIRC we don't and this was an artifact from testing :d

Reply:
(aka I'll double check and remove--thanks!)